lua/cmake/utils.lua

```diff
@@ -44,8 +54,13 @@ function utils.run(cmd, args, opts)
     on_stderr = vim.schedule_wrap(append_to_quickfix),
     on_exit = vim.schedule_wrap(function(_, exit_code)
       append_to_quickfix('Exited with code ' .. exit_code)
-      if exit_code == 0 and opts.on_success then
-        opts.on_success()
+      if exit_code == 0 then
+        if opts.on_success then
+          opts.on_success()
+        end
+      elseif config.quickfix_if_error then
+        go_to_quickfix()
+        vim.api.nvim_command('cbottom')
```

Comment:
Why are you calling `cbottom` additionally?

Reply:
otherwise it opens quickfix window on the first line. and opening at the bottom feels more natural to me. It is a matter of taste so I don't insist (`cnext` and `cprev` seems to be unaffected by opening on the first or on the last line)

Reply:
Oh, sounds reasonable.